ephios/core/management/commands/devdata.py

```diff
@@ -23,15 +23,15 @@ def create_objects():
 
     from django.contrib.auth.models import Group
 
-    volunteers = Group.objects.create(name=_("Volunteers"))
+    volunteers = Group.objects.create(name=gettext("Volunteers"))
```

Comment:
This was interesting: model char fields must get a str as value, not the proxy object. With the lazy version here, this later failed when `str(volunteers)` was called somewhere inside modellogging, because `__str__` did not return a `str` object, because `Group` simply returned the value of its name field.